docs/site-planning.md

```diff
@@ -1,44 +1,86 @@
-Site Planning
-=============
+Introduction
+--------
```

Comment:
I think we should retain the "Site Planning" h1 header: https://opensciencegrid.github.io/technology/documentation/style-guide/#headers